lib/1.4/utility.dml

```diff
@@ -1367,7 +1367,7 @@ template map_target is (connect, _qname) {
          Provides a shorthand to the API function `SIM_issue_transaction`.
```

Comment:
I suppose the motivation is that you may want to override the method to add extra atoms or whatever, and still be able to access stuff conveniently through read/write methods.

In order for users to understand this, you need to update documentation, maybe say that it's called by read/write methods and can be overridden.

Reply:
Ok, but if users are going to override it they'll need to read the source - documentation won't be enough.

Reply:
Unless you document it properly. This is a documented library, not just common code, so as soon as it's released, the responsibilities of the method are fixed, whether or not you document them (due to Hyrum's law). It's best to document those responsibilities, to make them explicit both for us and for users.

Reply:
I don't fully agree. If someone is likely to read the source anyway, duplicating information easily digested from the source in documentation seems unnecessary and inexact.

Anyway, I'll improve the docs :-)